contracts/peg/NonCustodialPSM.sol

```diff
@@ -333,6 +335,7 @@ contract NonCustodialPSM is
 
     /// @notice getter to return the maximum amount of Fei that could be purchased at once
     /// @return the maximum amount of Fei available for purchase at once through this PSM
+    // todo take out
```

Comment:
should it be taken out?

Reply:
no need to take this out. this function should be updated to properly account for the amount of assets sitting in the pcv deposit that this contract can draw from so that the UI can cleanly display things if this is going to be going on the mint/redeem page

Reply:
leaving it, won't hurt anything